<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ANOVAOneWay</class>
 <widget class="QWidget" name="ANOVAOneWay">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>606</width>
    <height>1167</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_4">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>6</number>
   </property>
   <item row="0" column="0">
    <widget class="QWidget" name="widget" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>573</width>
       <height>0</height>
      </size>
     </property>
     <widget class="AvailableFieldsListView" name="listAvailableFields">
      <property name="geometry">
       <rect>
        <x>12</x>
        <y>62</y>
        <width>166</width>
        <height>224</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>200</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>210</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="selectionMode">
       <enum>QAbstractItemView::ExtendedSelection</enum>
      </property>
     </widget>
     <widget class="QWidget" name="widget_3" native="true">
      <property name="geometry">
       <rect>
        <x>188</x>
        <y>62</y>
        <width>281</width>
        <height>451</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>1</verstretch>
       </sizepolicy>
      </property>
      <widget class="QLabel" name="label_main_fields">
       <property name="geometry">
        <rect>
         <x>58</x>
         <y>0</y>
         <width>124</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Dependent Variable</string>
       </property>
      </widget>
      <widget class="BoundListView" name="main_fields">
       <property name="geometry">
        <rect>
         <x>58</x>
         <y>24</y>
         <width>210</width>
         <height>31</height>
        </rect>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Maximum" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>210</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="selectionMode">
        <enum>QAbstractItemView::ExtendedSelection</enum>
       </property>
      </widget>
      <widget class="AssignButton" name="buttonAssign_main_fields">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>20</y>
         <width>50</width>
         <height>32</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>50</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="label_main_factor">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>70</y>
         <width>91</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Fixed Factors</string>
       </property>
      </widget>
      <widget class="AssignButton" name="buttonAssign_main_factor">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>100</y>
         <width>50</width>
         <height>32</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>50</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="BoundListView" name="main_factor">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>90</y>
         <width>213</width>
         <height>91</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
      </widget>
      <widget class="AssignButton" name="buttonAssign_main_factor_2">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>230</y>
         <width>50</width>
         <height>32</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>50</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="BoundListView" name="main_factor_2">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>220</y>
         <width>211</width>
         <height>91</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
      </widget>
      <widget class="QLabel" name="label_main_factor_2">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>200</y>
         <width>141</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Random Factors</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_main_factor_3">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>320</y>
         <width>141</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Covariates</string>
       </property>
      </widget>
      <widget class="BoundListView" name="main_factor_3">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>340</y>
         <width>213</width>
         <height>91</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
      </widget>
      <widget class="AssignButton" name="buttonAssign_main_factor_3">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>340</y>
         <width>50</width>
         <height>32</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>50</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="widget_6" native="true">
      <property name="geometry">
       <rect>
        <x>469</x>
        <y>12</y>
        <width>92</width>
        <height>112</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QPushButton" name="okButton">
         <property name="text">
          <string>OK</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_7">
         <property name="text">
          <string>Remove</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="widget_4" native="true">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>770</y>
        <width>573</width>
        <height>361</height>
       </rect>
      </property>
      <widget class="ExpanderButton" name="pushButton_3">
       <property name="geometry">
        <rect>
         <x>6</x>
         <y>0</y>
         <width>561</width>
         <height>32</height>
        </rect>
       </property>
       <property name="text">
        <string>Model</string>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
      </widget>
      <widget class="QWidget" name="widget_5" native="true">
       <property name="geometry">
        <rect>
         <x>206</x>
         <y>70</y>
         <width>281</width>
         <height>281</height>
        </rect>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>1</verstretch>
        </sizepolicy>
       </property>
       <widget class="QLabel" name="label_main_fields_2">
        <property name="geometry">
         <rect>
          <x>58</x>
          <y>0</y>
          <width>124</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Optional Terms</string>
        </property>
       </widget>
       <widget class="BoundListView" name="main_fields_2">
        <property name="geometry">
         <rect>
          <x>58</x>
          <y>24</y>
          <width>210</width>
          <height>131</height>
         </rect>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Maximum" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>210</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="selectionMode">
         <enum>QAbstractItemView::ExtendedSelection</enum>
        </property>
       </widget>
       <widget class="AssignButton" name="buttonAssign_main_fields_2">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>20</y>
          <width>50</width>
          <height>32</height>
         </rect>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>50</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>+</string>
        </property>
        <property name="icon">
         <iconset theme="go-next">
          <normaloff>:/icons/icons/1rightarrow.png</normaloff>:/icons/icons/1rightarrow.png</iconset>
        </property>
       </widget>
       <widget class="BoundListView" name="main_factor_5">
        <property name="geometry">
         <rect>
          <x>60</x>
          <y>180</y>
          <width>211</width>
          <height>91</height>
         </rect>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
       </widget>
       <widget class="QLabel" name="label_main_factor_5">
        <property name="geometry">
         <rect>
          <x>60</x>
          <y>160</y>
          <width>141</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Compulsory Terms</string>
        </property>
       </widget>
       <widget class="AssignButton" name="buttonAssign_main_fields_3">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>50</y>
          <width>50</width>
          <height>32</height>
         </rect>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>50</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>:</string>
        </property>
        <property name="icon">
         <iconset theme="go-next">
          <normaloff>:/icons/icons/1rightarrow.png</normaloff>:/icons/icons/1rightarrow.png</iconset>
        </property>
       </widget>
       <widget class="AssignButton" name="buttonAssign_main_fields_4">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>80</y>
          <width>50</width>
          <height>32</height>
         </rect>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>50</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>*</string>
        </property>
        <property name="icon">
         <iconset theme="go-next">
          <normaloff>:/icons/icons/1rightarrow.png</normaloff>:/icons/icons/1rightarrow.png</iconset>
        </property>
       </widget>
       <widget class="AssignButton" name="buttonAssign_main_fields_5">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>240</y>
          <width>50</width>
          <height>32</height>
         </rect>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>50</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>*</string>
        </property>
        <property name="icon">
         <iconset theme="go-next">
          <normaloff>:/icons/icons/1rightarrow.png</normaloff>:/icons/icons/1rightarrow.png</iconset>
        </property>
       </widget>
       <widget class="AssignButton" name="buttonAssign_main_fields_6">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>180</y>
          <width>50</width>
          <height>32</height>
         </rect>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>50</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>+</string>
        </property>
        <property name="icon">
         <iconset theme="go-next">
          <normaloff>:/icons/icons/1rightarrow.png</normaloff>:/icons/icons/1rightarrow.png</iconset>
        </property>
       </widget>
       <widget class="AssignButton" name="buttonAssign_main_fields_7">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>210</y>
          <width>50</width>
          <height>32</height>
         </rect>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>50</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>:</string>
        </property>
        <property name="icon">
         <iconset theme="go-next">
          <normaloff>:/icons/icons/1rightarrow.png</normaloff>:/icons/icons/1rightarrow.png</iconset>
        </property>
       </widget>
      </widget>
      <widget class="AvailableFieldsListView" name="listAvailableFields_2">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>70</y>
         <width>166</width>
         <height>224</height>
        </rect>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>200</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>210</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="selectionMode">
        <enum>QAbstractItemView::ExtendedSelection</enum>
       </property>
      </widget>
      <widget class="QCheckBox" name="checkBox_3">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>40</y>
         <width>431</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Full Factorial (don't pick this with bayesian stuff, it's too slow)</string>
       </property>
      </widget>
     </widget>
     <widget class="QCheckBox" name="checkBox">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>20</y>
        <width>101</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Frequentist</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox_2">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>20</y>
        <width>221</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Bayesian (ever so fashionable)</string>
      </property>
     </widget>
     <widget class="QWidget" name="widget_7" native="true">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>520</y>
        <width>573</width>
        <height>241</height>
       </rect>
      </property>
      <widget class="ExpanderButton" name="pushButton_4">
       <property name="geometry">
        <rect>
         <x>6</x>
         <y>0</y>
         <width>561</width>
         <height>32</height>
        </rect>
       </property>
       <property name="text">
        <string>Within Subjects</string>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
      </widget>
      <widget class="QWidget" name="widget_8" native="true">
       <property name="geometry">
        <rect>
         <x>206</x>
         <y>40</y>
         <width>281</width>
         <height>451</height>
        </rect>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>1</verstretch>
        </sizepolicy>
       </property>
       <widget class="QLabel" name="label_main_fields_3">
        <property name="geometry">
         <rect>
          <x>58</x>
          <y>0</y>
          <width>151</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Within subjects factors</string>
        </property>
       </widget>
       <widget class="BoundListView" name="main_fields_3">
        <property name="geometry">
         <rect>
          <x>58</x>
          <y>24</y>
          <width>210</width>
          <height>131</height>
         </rect>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Maximum" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>210</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="selectionMode">
         <enum>QAbstractItemView::ExtendedSelection</enum>
        </property>
       </widget>
       <widget class="AssignButton" name="buttonAssign_main_fields_8">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>20</y>
          <width>50</width>
          <height>32</height>
         </rect>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>50</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset theme="go-next">
          <normaloff>:/icons/icons/1rightarrow.png</normaloff>:/icons/icons/1rightarrow.png</iconset>
        </property>
       </widget>
      </widget>
      <widget class="AvailableFieldsListView" name="listAvailableFields_3">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>40</y>
         <width>166</width>
         <height>200</height>
        </rect>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>200</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>210</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="selectionMode">
        <enum>QAbstractItemView::ExtendedSelection</enum>
       </property>
      </widget>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ExpanderButton</class>
   <extends>QPushButton</extends>
   <header>widgets/expanderbutton.h</header>
  </customwidget>
  <customwidget>
   <class>BoundListView</class>
   <extends>QListView</extends>
   <header>widgets/boundlistview.h</header>
  </customwidget>
  <customwidget>
   <class>AssignButton</class>
   <extends>QPushButton</extends>
   <header>widgets/assignbutton.h</header>
  </customwidget>
  <customwidget>
   <class>AvailableFieldsListView</class>
   <extends>QListView</extends>
   <header>widgets/availablefieldslistview.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
